corehq/apps/case_search/tests/test_case_search_filters.py

```diff
@@ -116,3 +116,42 @@ def test_blank_case_search_parent(self):
             'parent/foo': ['', 'blackbeard'],
         }).get_ids()
         self.assertItemsEqual(actual, ['c4', 'c6', 'c8'])
+
+    def test_multivalued_property_case_search(self):
+        self._create_case_search_config()
+        cases = [
+            {'_id': 'c1', 'case_type': 'song', 'description': 'New York'},
+            {'_id': 'c2', 'case_type': 'song', 'description': 'Manchester'},
+            {'_id': 'c3', 'case_type': 'song', 'description': 'Manchester Boston'},
+        ]
+        self._assert_query_runs_correctly(
+            self.domain,
+            cases,
+            get_case_search_query(
+                self.domain,
+                ['song'],
+                {'x_commcare_description_config': "selected-any(description, 'New York Boston')"},
+            ),
+            None,
+            ['c1', 'c3']
+        )
+
+        self._create_case_search_config()
```

Comment:
This should be a separate test:

```suggestion
    def test_select_all_function(self):
        self._create_case_search_config()
```